src/solver/hydro/monthly/h2o_m_construire_les_variables.cpp

```diff
@@ -82,7 +81,18 @@ void H2O_M_ConstruireLesVariables(DONNEES_ANNUELLES& DonneesAnnuelles)
         Var++;
     }
 
-    for (int Pdt = 0; Pdt < NbPdt; Pdt++)
+    for (int Pdt = 0; Pdt < nbMonths; Pdt++)
+    {
+        CorrespondanceDesVariables.NumeroDeVariableOverflow[Pdt] = Var;
+        ProblemeLineairePartieVariable.Xmin[Var] = 0.0;
+        ProblemeLineairePartieVariable.Xmax[Var] = LINFINI;
+        ProblemeLineairePartieFixe.TypeDeVariable[Var] = VARIABLE_BORNEE_INFERIEUREMENT;
+        ProblemeLineairePartieVariable.AdresseOuPlacerLaValeurDesVariablesOptimisees[Var]
+          = &(DonneesAnnuelles.overflow[Pdt]);
```

Comment:
According to the Jira ticket, results for this variable are not needed

```suggestion
        ProblemeLineairePartieVariable.AdresseOuPlacerLaValeurDesVariablesOptimisees[Var]
          = nullptr;
```

Reply:
What if we want to print this new variable when activating **hydroDebug** ?

Reply:
Fair point